pyk/src/pyk/konvert/_kast_to_kore.py

```diff
@@ -249,6 +329,25 @@ def kflatmodule_to_kore(definition: KDefinition, kflatmodule: KFlatModule) -> Mo
     return Module(name=kflatmodule.name, sentences=(imports + kore_axioms))
 
 
+def _krule_att_to_kore(att_entry: AttEntry, kast_rule_vars: dict[str, list[KVariable]]) -> App:
```

Comment:
There's similar logic in `_module_to_kore`, maybe it's possible to extract that into it's own module and reuse it here.

Reply:
Hmmmm, I looked at implementing this as a `RulePass`, but I think that would create a circular dependency between `_kast_to_kore.py` and `_module_to_kore.py`. Maybe we should have `_outer_to_kore.py` and `_inner_to_kore.py`, since a lot of `_kast_to_kore.py` is converting `KInner -> Pattern`, and a lot of `_module_to_kore.py` is converting `KOuter -> KoreDefinition`. Then we could move `_krule_to_kore` to `_outer_to_kore.py`.

For now, I think I'll leave it as is.

Reply:
https://github.com/runtimeverification/k/issues/4690